package.json

```diff
@@ -56,9 +56,9 @@
     "@types/glob": "^7.2.0",
     "@types/googlemaps": "^3.30.16",
     "@types/lodash": "^4.14.136",
-    "@types/react": "^16.14.34",
-    "@types/react-datepicker": "^2.9.3",
-    "@types/react-dom": "^16.8.5",
+    "@types/react": "^17.0.2",
+    "@types/react-datepicker": "^4.10.0",
```

Comment:
Did you test any changes with datepicker?

Reply:
`react-datepicker` testing video:

https://user-images.githubusercontent.com/88361607/234486168-c529418f-8cae-4fcb-8f46-e20653966611.mov

Reply:
Nicely done.